docs/tools/cli/api.md

```diff
@@ -0,0 +1,150 @@
+---
+id: cli-api
+title: API
+pagination_label: CLI API
+sidebar_label: API
+sidebar_position: 10
+sidebar_class_name: cli-api
+keywords: ['cli', 'cli api', 'api']
+description: Learn how to use the CLI to make generic api calls in this guide.
+slug: /tools/cli/api
+tags: ['CLI']
+---
+
+Learn how to use the SailPoint CLI to make generic api calls in this guide.
+
+The `api` command makes it easy to call SailPoint APIs and parse the results using JSONPath queries if needed.
+
+- [get](#get-requests)
+- [post](#post-requests)
+- [patch](#patch-requests)
+- [put](#put-requests)
+- [delete](#delete-requests)
+
+## Get Requests
+
+Run this command to get a list of transforms using the v2025 API:
+
+```shell
+sail api get /v2025/transforms
+```
+
+You can filter the results using the `--jsonpath` flag or `-j` for short. The following command returns just the names of the transforms:
+
+```shell
+sail api get /v2025/transforms --jsonpath "$.*.name"
+```
+
+For better readability, you can pretty-print the response using the `--pretty` flag or `-p` for short:
+
+```shell
+sail api get /v2025/transforms --jsonpath "$.*.name" --pretty
+```
+
+### Headers
+
+To include headers with your API call—such as calling an experimental endpoint—use the `--headers` or `-H` flag:
+
+```shell
+sail api get /v2025/entitlements/ -H "X-SailPoint-Experimental:true"
+```
+
+### Query Parameters
+
+Use the `--query` or `-q` flag to provide query parameters.
+
+For example, to retrieve a list of entitlements owned by a specific identity:
```

Comment:
Change this "For example, run this command to retrieve a list of entitlements owned by a specific identity: